setup.py

```diff
@@ -23,11 +23,8 @@
     ],
     url="https://github.com/django-haystack/pysolr/",
     license="BSD",
-    install_requires=[
-        "requests>=2.9.1",
-        "setuptools",
-        "importlib_metadata; python_version<'3.8'",
-    ],
+    install_requires=["requests>=2.9.1", "setuptools"],
```

Comment:
requests 2.9.1 is pretty old (2015). I was wondering whether we might want to bump this to `requests>=2.32` since that's technically needed to get support for Python 3.14 (2.32.5) and if we don't pin the minor version that's a release which came out over a year ago (2.32.0 was May 2024).